src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random random = new Random();
+        int index = random.nextInt(DifferentColors.values().length);
+        DifferentColors color = DifferentColors.values()[index];
+        return color.toString();
```

Comment:
Consider using `color.name()` instead of `color.toString()` to get the string representation of the enum constant. This is a more reliable method as `toString()` can be overridden, which might lead to unexpected results.